dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -1,7 +1,10 @@
 class RubricsController < ApplicationController
   def edit
+    @rubric = Rubric.find_by(id: params[:id])
+    @lesson = Lesson.find_by(id: @rubric.lesson_id)
```

Comment:
I was trying to find an efficient way to pass around this data.  For a while I had a query param as well in the URL which also worked, but this seemed more efficient.  That being said, I looked in our code base for anywhere else that did anything similar to this and I couldn't find anything which gave me pause.

Reply:
What data do you mean, sorry? The lesson? You can just do `@lesson = @rubric.lesson` to get that (it's pretty much the same as what you wrote but more "rails-y")